### PR TITLE
add combobox and rbui v1

### PR DESCRIPTION
**Pull request by iseth**
this adds the combobox component and rbui gem and js package v1 alpha. 
This PR can be used as a reference on how to add the js to the phlex_ui / rbui mono-repo.

image

also add all available components and js controllers
image

### CODE REVIEW

**Comment by iseth** [comment]
tests should be passing when https://github.com/PhlexUI/phlex_ui/pull/73 gets merged

**Comment by iseth** [comment]
also maybe you could take a look at why the component breaks when I comment out the phlex_ui in the application.js file.
It seems like it has to do with the `setFloatingElement()` func.

File: app/views/docs/combobox_view.rb [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+      end
+      render Docs::ComponentsTable.new(components)
+    end
+  end
+
+  private
+
+  def components
+    [
+      Docs::ComponentStruct.new(name: "ComboboxController", source: "https://github.com/PhlexUI/phlex_ui_stimulus_pro/blob/main/controllers/command_controller.js", built_using: :stimulus),
```
Comment (cirdes) [inline app/views/docs/combobox_view.rb:56]:
```suggestion
      Docs::ComponentStruct.new(name: "ComboboxController", source: "https://github.com/PhlexUI/phlex_ui/blob/v1/lib/rbui/combobox/combobox_controller.js", built_using: :stimulus),
```

File: app/views/docs/combobox_view.rb [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+          end
+        RUBY
+      end
+      render Docs::ComponentsTable.new(components)
+    end
+  end
+
+  private
+
+  def components
```
Comment (cirdes) [inline app/views/docs/combobox_view.rb:54]: the components list should be updated

**Review by cirdes — CHANGES_REQUESTED** [review]
@iseth , please take a look at the CI and review the components' GitHub links.

**Pull request merged**